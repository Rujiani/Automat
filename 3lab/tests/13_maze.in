số nguyên step;

chu kỳ step = 0:50 bắt đầu
    nếu *_*=>x < exit_x + 1 && exit_x < *_*=>x + 1 &&
        *_*=>y < exit_y + 1 && exit_y < *_*=>y + 1 bắt đầu
            step = 50;
    đuôi


    { v_0; >_<; v_v; };
    { >_0; >_<; >_>; };
đuôi