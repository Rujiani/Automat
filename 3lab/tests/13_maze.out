[step 1] (0,0) -> (1,0)
[step 2] (1,0) -> (1,1)
[step 3] (1,1) -> (1,2)
[step 4] (1,2) -> (2,2)
[step 5] (2,2) -> (3,2)
[step 6] (3,2) -> (4,2)
[step 7] (4,2) -> (4,3)
[step 8] (4,3) -> (4,4)
finish
